model/schema/expression.yaml

```diff
@@ -30,115 +49,126 @@ classes:
     notes: >-
       Curators can provide several levels of evidence for an annotation, in an
       attempt to offer support the annotation schema at various MODs.
-      First, a curator can provide simply a reference (FB).
-      Second, a curator can provide a Figure within that reference (ZFIN).
-      Third, a curator can provide an ImagePane within some Figure (MGI).
+      A curator can provide a Figure within that reference (ZFIN).
+      Also, a curator can provide an ImagePane within some Figure (MGI).
       It's unclear which of these approaches will be chosen for future Alliance
       expression curation.
     slots:
-      - curie
-      - gene
+      - belongs_to_expression_experiment
       - when_expressed
       - where_expressed
-      - assay
-      - reagents
       - expression_qualifiers
-      - perturbed_expression_qualifiers
-      - perturbed
       - negated
       - uncertain
-      - primary_genetic_entities
-      - when_expressed_note
-      - where_expressed_note
-      - expression_annotation_note
-      - has_reference
-      - has_figure
+      - associated_with_figure    # this is often redundant
     slot_usage:
-      has_figure:
+      associated_with_figure:
         description: >-
-          The figure that supports the expression annotation.
+          The figure(s) that support(s) the expression annotation.
         notes: >-
+          This can seem redundant with ExpressionAnnotationImagePane.
+          But, sometimes we know the figure but don't have the image.
           Optional, as existing annotations from some MODs have no related
           images or figures.
         required: false
-        multivalued: false
+        multivalued: true
       has_reference:
         multivalued: false
 
+  TemporalContext:
```

Comment:
As a class, this would turn into a table of unique TemporalContexts - is this what we want?  Would a curator search for a unique combo of stage, age and temporal qualifiers?

Reply:
I don't know if a curator would search like this, but at FB, these combinations do get re-used often for different expression annotations, so it was an attempt to normalize the data.

Reply:
Does the temporal_qualifiers make this harder to reuse?  tracing down to temporal_qualifier_set, it seems like this should be at the ExpressionAnnotation level vs. the TemporalContext level?

Reply:
Looking at this again, I think the temporal qualifier slot should be removed. FlyBase is the only MOD to have temporal qualifiers, but it turns out there's only one such term ("precursor"), and it's only ever used to qualify anatomical terms: e.g., "precursor" of "atrium".
So I guess the question is if it's useful to have a TemporalContext entity that combines only stage and age. Perhaps not.